pkg/virt-launcher/virtwrap/access-credentials/access_credentials_test.go

```diff
@@ -100,13 +100,13 @@ var _ = Describe("AccessCredentials", func() {
 	})
 
 	It("should handle dynamically updating user/password with qemu agent", func() {
-
 		domName := "some-domain"
-		password := "1234"
 		user := "myuser"
-		base64Str := base64.StdEncoding.EncodeToString([]byte(password))
-		cmdSetPassword := fmt.Sprintf(`{"execute":"guest-set-user-password", "arguments": {"username":"%s", "password": "%s", "crypted": false }}`, user, base64Str)
-		mockConn.EXPECT().QemuAgentCommand(cmdSetPassword, domName).Return("", nil)
+		password := "1234"
+
+		mockConn.EXPECT().LookupDomainByName(gomock.Any()).Return(mockDomain, nil).AnyTimes()
```

Comment:
I would prefer having the control of how many times the lookup is executed.
That request can leak so we should be strict playing with it.
Thanks

Reply:
Sure I'll adjust it now.